Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonIntegratedToolBarManager.cs

```diff
@@ -40,7 +40,7 @@ public class KryptonIntegratedToolBarManager : Component
 
         private bool _flipButtonArray;
 
-        private bool _showIntegratedToolBar;
+        private bool? _allowFormIntegration;
```

Comment:
When is this ever going to be null ?
It will either be true or false (by default!)

Reply:
Reverted